.clippy.toml

```diff
@@ -2,5 +2,6 @@ upper-case-acronyms-aggressive = true
 
 # only intended to affect main rustls crate - need to allow disallowed-types in all other crates in Clippy CI job
 disallowed-types = [
+  { path = "portable_atomic_ptr::Arc", reason = "must use Arc from sync module to support configuration of Arc implementation used in this fork" },
```

Comment:
- used the wrong value for `path` here - should have used this: `portable_atomic_util::Arc`
- also need to add entry with `path = "alloc::sync::Arc"` to avoid this with no `std` feature enabled